docs/core/metrics.md

```diff
@@ -78,6 +78,19 @@ You can create metrics using `add_metric`, and you can create dimensions for all
 ???+ warning "Warning: Do not create metrics or dimensions outside the handler"
     Metrics or dimensions added in the global scope will only be added during cold start. Disregard if you that's the intended behavior.
 
+### Adding high-resolution metrics
```

Comment:
Need your help here @heitorlessa! We must improve this topic or even add more information to the entire documentation.

Reply:
Suggestion: Add a new banner `When is it useful?` to explain to newcomers the common use cases for this feature like telemetry, timeseries, incident management, etc.

Reply:
I added new information and I think it's good now.